src/components/Roadmap/RoadmapActionCard.tsx

```diff
@@ -31,53 +27,13 @@ const RoadmapActionCard: React.FC<IProps> = ({
   description,
   buttonText,
 }) => {
-  const data = useStaticQuery(graphql`
-    query RoadmapActionCard {
-      futureProofing: file(relativePath: { eq: "roadmap/roadmap-future.png" }) {
-        childImageSharp {
-          gatsbyImageData(
-            width: 400
-            height: 260
-            layout: CONSTRAINED
-            placeholder: BLURRED
-            quality: 100
-          )
-        }
-      }
-      scaling: file(relativePath: { eq: "roadmap/roadmap-transactions.png" }) {
-        childImageSharp {
-          gatsbyImageData(
-            height: 260
-            layout: CONSTRAINED
-            placeholder: BLURRED
-            quality: 100
-          )
-        }
-      }
-      security: file(relativePath: { eq: "roadmap/roadmap-security.png" }) {
-        childImageSharp {
-          gatsbyImageData(
-            width: 380
-            height: 260
-            layout: CONSTRAINED
-            placeholder: BLURRED
-            quality: 100
-          )
-        }
-      }
-      userExperience: file(relativePath: { eq: "roadmap/roadmap-ux.png" }) {
-        childImageSharp {
-          gatsbyImageData(
-            width: 380
-            height: 260
-            layout: CONSTRAINED
-            placeholder: BLURRED
-            quality: 100
-          )
-        }
-      }
-    }
-  `)
+  const data = {
+    futureProofing: "/roadmap/roadmap-future.png",
+    scaling: "/roadmap/roadmap-transactions.png",
+    security: "/roadmap/roadmap-security.png",
+    userExperience: "/roadmap/roadmap-ux.png",
```

Comment:
Yeah, I pushed an update around the time you were reviewing this that added back some explicit height/width numbers, based on the graphql queries we were using on Gatsby, which does fix it, but is manual. 

I just switched to the approach you showed and it removed the need to hard-code these ❤️ 👍 

---

@pettinarip @nhsz Side note... things in `public` are easy to reference in the DOM, but slightly messier to import. We can simplify the imports using this syntax for example:

```ts
import userExperience from "@/../public/roadmap/roadmap-ux.png"
```

If we add the following to `compilerOptions.paths` in `tsconfig.json` we could use `@/public/` to simplify further any imports from the public folder:

```ts
`"@/public/*": ["./public/*"]`
```

^ Let me know if you have thoughts on this... easy patch that could be applied separately if we want it. Going to use `@/../public/` format for now to avoid using relative paths.

Reply:
I'm down with adding that alias.

However, shouldn't the new import syntax be
- `"@/public/roadmap/roadmap-ux.png"` instead of
- `"@/../public/roadmap/roadmap-ux.png"`?

Reply:
Yepp!.... but only IF we add that line to the `tsconfig` =)... If we're good with that pattern, I'll update `tsconfig` and switch `@/../public/` to `@/public/`

Reply:
Just pushed a commit to update this using `@/public`